tests/src/readytorun/r2rdump/R2RDumpTest.csproj

```diff
@@ -11,10 +11,10 @@
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
     <CoreClrDir>..\..\..\..\..\..\</CoreClrDir>
     <R2RDumpCommand>$(CoreClrDir)bin\Product\$(BuildOS).$(BuildArch).$(BuildType)\netcoreapp2.0\R2RDump.dll</R2RDumpCommand>
-    <DotnetToolCommand>$(CoreClrDir)Tools\dotnetcli\dotnet</DotnetToolCommand>
+    <DotnetToolCommand>dotnet</DotnetToolCommand>
     <BashCoreClrDir>../../../../../../</BashCoreClrDir>
     <BashR2RDumpCommand>$(BashCoreClrDir)bin/Product/$(BuildOS).$(BuildArch).$(BuildType)/netcoreapp2.0/R2RDump.dll</BashR2RDumpCommand>
-    <BashDotnetToolCommand>$(BashCoreClrDir)Tools/dotnetcli/dotnet</BashDotnetToolCommand>
+    <BashDotnetToolCommand>dotnet</BashDotnetToolCommand>
```

Comment:
Could we just use DotnetToolCommand now?

Reply:
arcade install dotnet to the path.  When InitBuildTools is executed.  I assumed it would already bo on the path.